server/middleware/validationMiddleware.ts

```diff
@@ -12,6 +12,13 @@ export const buildErrorSummaryList = (array: fieldErrors) => {
   }))
 }
 
+export const findError = (errors: fieldErrors, fieldName: string) => {
+  if (!errors) {
+    return null
+  }
+  return errors[fieldName]?.[0]
```

Comment:
I think this needs to be
```
return {
    text: errors[fieldName]?.[0],
  }
```
as the `errorMessage` nunjucks property expects `{ text: "error message" }`

Reply:
Fixed